Source/WebCore/platform/CaretAnimation.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2022 Apple Inc. All rights reserved.
```

Comment:
Since we're copying the code from FrameSelection.cpp, we need to inherit the copyright information from that file instead.